### ISSUE TITLE
Bug: php spark serve

### ISSUE THREAD

**Issue by seventechsolucoes** [codeigniter4/CodeIgniter4#2784]
None of the installations are running the command php spark serves, it is returning the following errors.

![Screenshot_1](https://user-images.githubusercontent.com/6888653/78080493-5a849a00-737c-11ea-8483-f2c55e39895f.png)

**Afeter composer update**

![next composer update](https://user-images.githubusercontent.com/6888653/78080555-7c7e1c80-737c-11ea-8057-db6712b4d8e7.png)

**Comment by lonnieezell** [comment]
Well, my search-fu is failing me tonight, but I'm know that I've seen this come up on the forums before, and there was a setting for the server under Windows that was the issue. Please post there. 

ATM I don't believe this is a bug.

**Comment by seventechsolucoes** [comment]
I downloaded these links
https://github.com/codeigniter4/framework/releases
https://github.com/codeigniter4/codeigniter4/releases

And I also tried it via composer

composer create-project codeigniter4 / appstarter --no-dev

I run the following commands for installation via composer

1 - php builds release
2 - composer update

**Comment by lonnieezell** [comment]
Ok, thanks. We will look into it.

**Comment by seventechsolucoes** [comment]
@lonnieezell

I found the problem, it's nothing with the framework, I was trying to run spark on this path:

C:\Users\guilh\OneDrive\Área de trabalho\framework-4.0.2

The namespaces failed in "Desktop", coincidentally when I tested on ubuntu the path was similar:

/home/guilherme/Área de trabalho

So just put it in a folder with a path without spaces c:\ci4 for example

**Comment by MGatner** [comment]
We've fixed this one a few times (e.g. #1880). I think @michalsn took a look at it somewhat recently. Michal do you recall?

**Comment by michalsn** [comment]
The issue here is not "spaces" but the accented character `Á` - and in general every other accented character.

In the Autoloader file, we wipe out all non-standard characters: https://github.com/codeigniter4/CodeIgniter4/blob/develop/system/Autoloader/Autoloader.php#L388, so that's why the path to all files is wrong.

I'm not really sure how we should handle this but... I probably would leave it as it is since in another case we would need to support every character.

So in short. We may need a note somewhere that we only support characters deemed safe for POSIX names.

**Comment by samsonasik** [comment]
It probably related due to mbstring not required in composer.json that can't detect character `Á`. In other forum in facebook, I got people ask about it as well, and solved by install mbstring extension.

**Comment by samsonasik** [comment]
I created PR #2803 for it.

**Comment by 2006878** [comment]
Estou com problemas para rodar localmente um projeto CodeIgniter, aparece o seguinte erro: 

PS C:\Users\User\Desktop\Projetos\GitHub\appstarter> php spark serve

CodeIgniter v4.1.4 Command Line Tool - Server Time: 2021-11-06 08:44:04 UTC-05:00

CodeIgniter development server started on http://localhost:8080 Press Control-C to stop. [Sat Nov 6 10:44:12 2021] PHP 8.0.3 Development Server (http://localhost:8080) started [Sat Nov 6 10:44:13 2021] [::1]:50258 Accepted [Sat Nov 6 10:44:13 2021] [::1]:50259 Accepted [Sat Nov 6 10:44:14 2021] PHP Warning: Unknown: Failed to open stream: No such file or directory in Unknown on line 0 [Sat Nov 6 10:44:14 2021] PHP Fatal error: Failed opening required 'C:\Users\User\Desktop\Projetos\GitHub\appstarter\vendor\codeigniter4\framework\sys tem\Commands\Server/rewrite.php' (include_path='.;C:\php\pear') in Unknown on line 0 [Sat Nov 6 10:44:14 2021] [::1]:50259 Closing

Alguem conseguiria me dar uma luz? Estou iniciando no CodeIgniter agora.

**Comment by samsonasik** [comment]
@2006878 that's unrelated, your rewrite.php seems removed by antivirus, you can exclude the full path of file in your antivirus settings, also, please use english.

**Comment by 2006878** [comment]
@samsonasik On installation this file did not come. But I found it in a repository here on Github I copied and pasted it into the code and it ran. Thanks for listening!
